packages/kbn-react-field/src/field_icon/__snapshots__/field_icon.test.tsx.snap

```diff
@@ -275,6 +275,16 @@ exports[`FieldIcon renders known field types shape is rendered 1`] = `
 />
 `;
 
+exports[`FieldIcon renders known field types sparse_vector is rendered 1`] = `
+<EuiToken
+  aria-label="sparse_vector"
+  className="kbnFieldIcon"
+  iconType="tokenDenseVector"
```

Comment:
Do we want to use `tokenDenseVector` or should we create a new token for sparse vector?